src/components/test/ReplyBlock.test.tsx

```diff
@@ -0,0 +1,82 @@
+import { keccak_256 } from "js-sha3";
+import { shallow, mount } from "enzyme";
+import ReplyBlock from "../ReplyBlock";
+import { componentWithStore, createMockStore } from "../../test/testhelpers";
+import { getPrefabFeed } from "../../test/testFeeds";
+import { getPrefabProfile } from "../../test/testProfiles";
+import { waitFor } from "@testing-library/react";
+import * as sdk from "../../services/sdk";
+
+const profile = getPrefabProfile(0);
+const feed = getPrefabFeed();
+const initialState = { user: { profile }, feed: { feed } };
+const store = createMockStore(initialState);
+
+beforeAll(() => {
```

Comment:
I think this needs to be inside a describe block right?

Reply:
good catch. thanks :)